AutomatedTests/cypress/ExampleRequests/sync_xml_responses.js

```diff
@@ -0,0 +1,9 @@
+const getSyncCivilAppearance = '<?xml version="1.0" encoding="utf-8" ?><env:Envelope xmlns:env="http://www.w3.org/2003/05/soap-envelope"><env:Header/><env:Body><ser-root:getSyncCivilAppearanceResponse xmlns:ns4="http://courts.gov.bc.ca/XMLSchema/PCSS/1.0.0" xmlns:ser-root="http://courts.gov.bc.ca/xml/ns/pcss/civil/v1" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"><getSyncCivilAppearanceResponse><ns4:getSyncCivilAppearanceResponse><ns4:ResponseCd>1</ns4:ResponseCd><ns4:ResponseMessageTxt>No change detected for any future civil appearance</ns4:ResponseMessageTxt></ns4:getSyncCivilAppearanceResponse></getSyncCivilAppearanceResponse></ser-root:getSyncCivilAppearanceResponse></env:Body></env:Envelope>'
```

Comment:
It would be a good idea naming the file with post-fix V1 (same to the responses.js)